tardis/adapters/batchsystems/htcondor.py

```diff
@@ -172,14 +172,13 @@ async def get_resource_ratios(self, drone_uuid: str) -> Iterable[float]:
         await self._htcondor_status.update_status()
         try:
             htcondor_status = self._htcondor_status[drone_uuid]
-        except KeyError:
-            return {}
-        else:
             return (
                 float(value)
                 for key, value in htcondor_status.items()
                 if key in self.ratios.keys()
             )
+        except (KeyError, ValueError):
+            return {}
```

Comment:
I do not remember, why I put a `dict` here? Could you change it to an empty list or tuple instead, please?

Reply:
Ah, I did so already :D Great that you also recognised this!

Reply:
Yeah! I have a rough idea why this was `dict`, however it survived a refactoring and is actually not needed anymore. ;-)